tmt/base.py

```diff
@@ -2405,6 +2406,7 @@ class Story(
         'id',
         'priority',
         'description',
+        'contact',
```

Comment:
Seems that similar should be done for `Plan` as well, as `tmt plan show` does not show contact. Please, extend test for `tmt plan show` and ensure that contact is show as expected. Thanks.

Reply:
Thanks!